xml/System.ServiceModel/WSDualHttpBinding.xml

```diff
@@ -711,7 +711,7 @@
    
   
 ## Examples  
- The following examples show how to get the <xref:System.ServiceModel.WSDualHttpBinding.System.ServiceModel.Channels.IBindingRuntimePreferences.ReceiveSynchronously%2A> value.  
+ The following examples show how to get the <xref:System.ServiceModel.WSDualHttpBinding.System%23ServiceModel%23Channels%23IBindingRuntimePreferences%23ReceiveSynchronously%2A> value.  
```

Comment:
Oh ... I see what you mean ... https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.wsdualhttpbinding.maxreceivedmessagesize?view=netframework-4.7#System_ServiceModel_WSDualHttpBinding_MaxReceivedMessageSize

I'm far too trusting! lol